web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -437,6 +437,10 @@ public function work(string $wid): array {
       'type-text' => $this->t('Type', [], ['context' => 'Work Page']),
       'we-have-shopped-text' => $this->t('In stock:', [], ['context' => 'Work Page']),
       'you-have-borrowed-text' => $this->t('You have borrowed', [], ['context' => 'Work Page']),
+      'share-on-facebook-text' => $this->t('Share on Facebook', [], ['context' => 'Work Page']),
```

Comment:
Just to make sure - these buttons only appear on the work page, right?

Reply:
For now, yes